src/content.js

```diff
@@ -149,14 +149,24 @@ export function drawKey(element, keyMap) {
 }
 
 export function drawDK(element, keyMap, deadKey) {
```

Comment:
Do we need the `export` keyword here ? (= do we really want to export this function ?)

Reply:
I’d say yes, it is used by `x-keyboard.js` : https://github.com/fabi1cazenave/x-keyboard/pull/29/files#diff-6a4b48a52a7703e5dc74b3a02c11978ba66663dda879d004c5767c56f1c26bf8R184-R186

Reply:
Duh. Indeed.